config/routes.rb

```diff
@@ -3,8 +3,14 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
-  resources :users do
-    resources :posts, only: [:index, :show], controller: 'posts'
-  end
   
-end
+  get '/users/:user_id/posts(.:format)', to: 'posts#index'
+  get  '/users/:user_id/posts(.:format)', to:  'posts#index'
+  get  '/users/:user_id/posts/:id(.:format)', to: 'posts#show'
+  get  '/users(.:format)', to: 'users#index'
+  get  '/users/:id(.:format)', to: 'users#show'
```

Comment:
- [ ] Kindly remove the duplicated route as now there exist 5 routes.
![image](https://user-images.githubusercontent.com/24703171/221596343-d038b147-a58c-4b5f-acff-e9b586758b21.png)